docs/installation.rst

```diff
@@ -8,38 +8,37 @@ Installation
 
 .. code-block:: bash
 
-    $ git clone https://github.com/BeyondTheClouds/enos
-
-You should also choose to go with a virtualenv. Please refer to the `virtualenv
-<https://virtualenv.pypa.io/en/stable/>`_ documentation for further information.
+    $ pip install enos
```

Comment:
Is it true. I mean, you could change your reservation.yaml to deploy the one in development.